src/Runtime/ParentRuntime.php

```diff
@@ -27,6 +27,9 @@ class ParentRuntime
 
     protected static $myPid = null;
 
+    /** @var string */
+    protected static $executable = 'php';
```

Comment:
Maybe we should default this value to the binary located by https://github.com/symfony/process/blob/v4.4.2/PhpExecutableFinder.php?

Reply:
By which, I mean, we default it to `null`, and have a `getExecutable()` method that falls back to using the executable finder.